certora/harness/MorphoHarness.sol

```diff
@@ -22,6 +22,14 @@ contract MorphoHarness is Morpho {
         return idToMarketParams[id];
     }
 
+    function toMarket(Id id) external view returns (Market memory) {
+        return market[id];
+    }
+
+    function toPosition(Id id, address borrower) external view returns (Position memory) {
+        return position[id][borrower];
+    }
```

Comment:
```suggestion
    function position(Id id, address borrower) external view override(Morpho) returns (Position memory) {
        return position[id][borrower];
    }
```

Reply:
Implemented in https://github.com/morpho-org/morpho-blue/pull/700/commits/b67e25af8da19d2b466c6f88d52ad769222ee518. Using naming convention suggested by Quentin to avoid name clashed and shadowing.